.env.example

```diff
@@ -19,8 +19,10 @@ JWT_SECRET=thishastobesecret13213
 JWT_EXPIRY_TIME=7 days
 
 # ------- OPTIONAL ------- #
+WEB3_BACKUP=http://localhost:8545
```

Comment:
Since ethers fallback provider accepts an array of providers, would be to allow user to specify N nodes formatted as

`WEB3=node1url;node2url;...nodeNurl`

Then use the order of this config to assign the prioroties